src/PayloadListener/Extensions/ValidationExtensions.cs

```diff
@@ -25,6 +25,20 @@ public static bool IsValid(this WorkflowRequestEvent workflowRequestMessage, out
             return valid;
         }
 
+        public static bool IsValid(this TaskUpdateEvent taskUpdateMessage, out IList<string> validationErrors)
+        {
+            Guard.Against.Null(taskUpdateMessage, nameof(taskUpdateMessage));
+
+            validationErrors = new List<string>();
+
+            var valid = true;
+
+            valid &= !taskUpdateMessage.WorkflowInstanceId.ValueIsNullOrWhiteSpace(taskUpdateMessage.GetType().Name, validationErrors);
```

Comment:
calling `taskUPdateMessage.Validate()` checks for null or white space I believe.

Reply:
Good shout, didn't know we had that, I have migrated to use that.